package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@types/hast": "^2.0.0",
     "extend": "^3.0.0",
+    "hast-util-is-element": "^2.1.3",
```

Comment:
30 seconds to late, but I just noticed that all other deps are listed in their base major version. Should this also be "^2.0.0"?

Reply:
Perfect, that's what I figured. Didn't want it to be missed. 👍